recipes/pysbatch/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pysbatch" %}
+{% set version = "0.1.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/luptior/pysbatch/archive/v{{ version }}.zip
+  sha256: 466195724e6e272557f54ebd36680451b1756a390c086a2b70384dfb36f2620d
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.5
+  run:
+    - python >=3.5
+
+test:
+  imports:
+    - pysbatch
+
+about:
+  home: https://github.com/luptior/pysbatch
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: 'Submit slurm cluster job(Sbatch) inside python and avoid shell script.'
+
+  description: |
+    Submit(sbatch) slurm cluster job inside python and avoid shell script for complicated pipeline jobs. 
+    For sbatch options, now only supports job name, memory size(in GBs), time limit(in days), dependency 
+    and ouput file. But you can use add_option parameter to add more.
+  doc_url: https://github.com/luptior/pysbatch/blob/master/README.md
+  dev_url: https://github.com/luptior/pysbatch
+
+extra:
+  recipe-maintainers:
+    - Luptior
```

Comment:
I think your account in ``github`` is with the lower case ``L`` right?
So, I think it should be
```
- luptior
```